content/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -1,42 +1,52 @@
 It handles retrieval and propagation of blocks and transactions (messages), and
 thus is in charge of distributed state replication.
-**This process is security critical -- problems here can be catastrophic to the
-operation of a blockchain.**
+As such, this process is security critical -- problems with state replication can have severe consequences to the
+operation of a blockchain.
 
-## What is ChainSync?
+## ChainSync Overview
 
 `ChainSync` is the protocol Filecoin uses to synchronize its blockchain. It is
 specific to Filecoin's choices in state representation and consensus rules,
 but is general enough that it can serve other blockchains. `ChainSync` is a
 group of smaller protocols, which handle different parts of the sync process.
 
+Chain synchronisation generally applies to three main processes:
+1. Chain synchronization when a node first joins the network and needs to get to the current state before validating or extending the chain.
+2. Chain synchronisation when a node has fell out of sync, e.g., due to a brief disconnection.
+3. Continuous chain synchronisation to keep up with the latest messages and blocks.
+
+There are three main protocols used to achieve synchronisation for these three processes.
+- `GossipSub` is the libp2p pubsub protocol used to propagate messages and blocks. It is mainly used in the third process above when a node needs to stay in sync with new blocks being produced and propagated.
+- `BlockSync` is used to synchronise specific parts of the chain, that is from and to a specific height.
+- `Bitswap` is used to request and receive blocks, when a node is synchonized but GossipSub has failed to deliver some blocks to a node.
+- `GraphSync` can be used to fetch parts of the blockchain as a more efficient version of `Bitswap`.
+
+Filecoin nodes are libp2p nodes, and therefore may run a variety of other protocols. As with anything else in Filecoin, nodes MAY opt to use additional protocols to achieve the results. That said, nodes MUST implement the version of `ChainSync` as described in this spec in order to be considered implementations of Filecoin. 
+
 ## Terms and Concepts
 
 - `LastCheckpoint` the last hard social-consensus oriented checkpoint that `ChainSync` is aware of.
   This consensus checkpoint defines the minimum finality, and a minimum of history to build on.
   `ChainSync` takes `LastCheckpoint` on faith, and builds on it, never switching away from its history.
 - `TargetHeads` a list of `BlockCIDs` that represent blocks at the fringe of block production.
   These are the newest and best blocks `ChainSync` knows about. They are "target" heads because
-  `ChainSync` will try to sync to them. This list is sorted by "likelihood of being the best chain" (eg for now, simply `ChainWeight`)
+  `ChainSync` will try to sync to them. This list is sorted by "likelihood of being the best chain". At this point this is simply realized through `ChainWeight`.
```

Comment:
`ChainSync` (a term I insist doesn't exist in the code) seems to refer to a process while `ChainWeight` is a parameter of our current chain. They may be related but saying one is used to achieve the other might be a bit of a stretch that may require more explanatory text adapting the two terms together.